File: airbyte-commons-server/src/main/java/io/airbyte/commons/server/errors/NotFoundException.java
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.server.errors;

/**
 * The file could not be found.
 */
public class NotFoundException extends KnownException {

  public NotFoundException(final String message) {
    super(message);
  }

  public NotFoundException(final String message, final Throwable cause) {
    super(message, cause);
  }

  @Override
  public int getHttpCode() {
    return 404;
  }

}

File: airbyte-commons-server/src/main/kotlin/io/airbyte/commons/server/errors/NotFoundException.kt
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.server.errors

/**
 * The file could not be found.
 */
class NotFoundException : KnownException {
  constructor(message: String?) : super(message)

  constructor(message: String?, cause: Throwable?) : super(message, cause)

  override fun getHttpCode(): Int = 404
}
